src/main/resources/application.properties

```
@@ -1,14 +1,21 @@
-spring.jpa.hibernate.ddl-auto=none
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5432/shareit
+spring.datasource.username=dbuser
+spring.datasource.password=12345
+
+spring.jpa.hibernate.ddl-auto=create-drop
```

Еще стоит добавить диалект

```suggestion
spring.jpa.hibernate.ddl-auto=create-drop
spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
```